modules/tensor_mechanics/test/tests/beam/dynamic/dyn_euler_small_added_mass2.i

```diff
@@ -0,0 +1,347 @@
+# Test for small strain euler beam vibration in y direction
+
+# An impulse load is applied at the end of a cantilever beam of length 5ft (60 in).
+# The beam is massless with a lumped mass at the end of the beam of 5000 lb
+# The properties of the cantilever beam are as follows:
+# E = 1e7 and I = 120 in^4
+# Assuming a square cross section A = sqrt(12 * I) = 37.95
+# Shear modulus (G) = 3.846e6
+# Shear coefficient (k) = 1.0
+# Cross-section area (A) = 1.0
```

Comment:
There is another line above with cross-section area

Reply:
I started off with the cross section being 1.0, but realized I needed to calculate it from the value of I given in this problem. I would just delete the line with A = 1.0.

Reply:
I will approve the merge request and then make this change along with the updates to the documentation.